src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -6,19 +6,20 @@
   position: absolute;
   background: $primary;
   top: 66px;
-  left: 50%;
+  // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
+  left: 40%;
```

Comment:
Is it possible to set a variable for the sidebar/map %'s, and then calculate the value here by using that variable divide 2?

Reply:
I think yes, should take not much time, but the sidebar's width/logic will change in https://github.com/sparkletown/sparkle/pull/1074

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1156/commits/d46487a0e6597ec71501e136b1822c8586897a0b